nuget/AspNetCore.nuspec

```diff
@@ -6,10 +6,10 @@
     <version>$VERSION$</version>
     <authors>Microsoft</authors>
     <owners>Microsoft</owners>
-    <licenseUrl>http://www.microsoft.com/web/webpi/eula/net_library_eula_ENU.htm</licenseUrl>
```

Comment:
It's not clear to me whether apache or MIT is correct, but either way, this URL is wrong.

Reply:
ANCM and IISIntegration used different licenses (ANCM == MIT, IISIntegration == Apache). I added both to the LICENSE.txt file after discussing with @Eilon . We may want a better solution than this.

Reply:
This package is deliberately different from other packages in ASP.NET Core, so we shouldn't change what's already there (and vetted). However, we *must* change the URLs to all be HTTPS.

Reply:
ok, reverted to original URL (plus https)